flume/conf/krb5.conf

```diff
@@ -0,0 +1,17 @@
+[libdefaults]
```

Comment:
It is automatically copied since it is placed at `conf` folder:

```
function copy_cygnus_conf() {
	_logStage "######## Copying cygnus template config files to destination config directory... ########"
	rm -rf {RPM_SOURCE_DIR}/config 
	mkdir -p ${RPM_SOURCE_DIR}/config
	for file in $(ls ${BASE_DIR}/conf)
	do
		file=$(basename ${file})
		renamed_file=${file%.template}
		cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/${renamed_file}
	done
}
```

Nevertheless, I've changed its name to `krb5.c